src/modules/attitude_estimator_ekf/attitude_estimator_ekf_main.cpp

```diff
@@ -595,6 +595,15 @@ const unsigned int loop_interval_alarm = 6500;	// loop interval in microseconds
 					memcpy(&att.R[0], &R.data[0][0], sizeof(att.R));
 					att.R_valid = true;
 
+					// untested
```

Comment:
The results in QGC looked weird, but it could be the MAVLink conversion bug. I ran out of time to finish this completely.